firefox-ios/firefox-ios-tests/Tests/XCUITests/PhotonActionSheetTests.swift

```diff
@@ -26,10 +26,12 @@ class PhotonActionSheetTests: BaseTestCase {
         // Remove pin
         cell.press(forDuration: 2)
         app.tables.cells.otherElements[StandardImageIdentifiers.Large.pinSlash].tap()
-
         // Check that it has been unpinned
-        // Adding sleep as an workaround until https://github.com/mozilla-mobile/firefox-ios/issues/22323 is fixed
-        sleep(5)
```

Comment:
great, thanks for removing this sleep

Reply:
removing this sleep will cause testPinToShortcuts test to fail, because the issue from the comment reproduces on simulator, but not on a real device

Reply:
wrote the comment, before seeing the workaround using  waitForNoExistence, if this works i am ok with it.

Reply:
https://github.com/user-attachments/assets/82432b66-9785-423e-ac0a-9bd5e5b4163d